ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ReleasesSample.cs

```diff
@@ -485,6 +485,47 @@ public void DeleteReleaseDefinition()
             releaseClient.DeleteReleaseDefinitionAsync(project: projectName, definitionId: newlyCreatedReleaseDefinitionId).SyncResult();
 
         }
+        
+        [ClientSampleMethod]
+        public IEnumerable<Deployment> ListAllDeploymentsForASpecificReleaseDefinitionId()
+        {
+            string projectName = ClientSampleHelpers.FindAnyProject(this.Context).Name;
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient2 releaseClient = connection.GetClient<ReleaseHttpClient2>();
+
+            var releaseDefinitions = releaseClient.GetReleaseDefinitionsAsync(project: projectName).Result;
```

Comment:
Avoid `var` in samples (people will be trying to learn from the samples and knowing the actual type of `releaseDefinitions` is important)